_docs/language-support-backup.md

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
This copyright notice appears in the preview.
Please verify whether this style of comment works in markdown.